src/components/ApplicationGroup/Details/EnvCIDetails/EnvCIDetails.tsx

```diff
@@ -24,58 +24,51 @@ export default function EnvCIDetails({ filteredApps }: AppGroupDetailDefaultType
     const [triggerHistory, setTriggerHistory] = useState<Map<number, History>>(new Map())
     const [fullScreenView, setFullScreenView] = useState<boolean>(false)
     const [hasMoreLoading, setHasMoreLoading] = useState<boolean>(false)
-    const [initDataResults, setInitResult] = useState<CIConfigListType>()
     const [pipelineList, setPipelineList] = useState<CiPipeline[]>([])
     const [ciGroupLoading, setCiGroupLoading] = useState(false)
+    const [securityModuleInstalled, setSecurityModuleInstalled] = useState(false)
+    const [blobStorageConfigured, setBlobStorageConfigured] = useState(false)
 
     useEffect(() => {
         try {
             setCiGroupLoading(true)
-            getCIConfigList(envId).then((result) => {
-                setInitResult(result)
+            getCIConfigList(envId, filteredAppIds).then((result) => {
+                if (result?.pipelineList.length) {
+                    const _filteredPipelines = []
+                    let nonWebhookCIExist = false
+                    let selectedPipelineExist = false
+                    result.pipelineList.forEach((pipeline) => {
+                        _filteredPipelines.push(pipeline)
+                        nonWebhookCIExist = true
```

Comment:
is this expected? If yes then it'll always go in if branch on line 46

Reply:
Done